deploy/Dockerfile

```diff
@@ -27,6 +27,8 @@ RUN make -C /go/src/github.com/awslabs/amazon-ecr-credential-helper linux-amd64
 # ACR docker credential helper
 ADD https://aadacr.blob.core.windows.net/acr-docker-credential-helper/docker-credential-acr-linux-amd64.tar.gz /usr/local/bin
 RUN tar -C /usr/local/bin/ -xvzf /usr/local/bin/docker-credential-acr-linux-amd64.tar.gz
+# Blank out the credentials file
+RUN echo "{}" > /root/.docker/config.json
```

Comment:
I believe a better approach would be to remove the line `RUN docker-credential-gcr configure-docker` which is the line that adds .docker/config.json

Reply:
Can't do that, as then kaniko blows up when it tries to read a non-existent config.json file. :(

[ Been there, done that, got the scars to prove it. ]

Reply:
got it, thanks for that extra info. @tejal29 your thoughts?